cli/azd/.vscode/cspell-azd-dictionary.txt

```diff
@@ -132,6 +132,7 @@ swacli
 Syncer
 teamcity
 testdata
+tmpl
```

Comment:
make this an exception just for `[kubectl.go](https://github.com/Azure/azure-dev/pull/2455/files#diff-e0c8b9bdf6f4944a3066ab25f9f73a2ecafe21f84a006a15fe69874d260e8164)`, It does not make to see this all over azd

Reply:
Use this file: https://github.com/Azure/azure-dev/blob/main/cli/azd/.vscode/cspell.yaml#L32

`tmpl` should be ok just within `kubectl.go` file. From any other azd file, we don't want to allow this world, right?